apps/tools/src/fund.command.ts

```diff
@@ -0,0 +1,183 @@
+import { DisbursementSummary, FundingService } from '@app/blockchain/funding.service'
+import { networkConfig, NetworkConfig } from '@app/utils/config/network.config'
+import { Inject } from '@nestjs/common'
+import BigNumber from 'bignumber.js'
+import commander from 'commander'
+import { Command, Console, createSpinner } from 'nestjs-console'
+import { Logger } from 'nestjs-pino'
+import { TransactionReceipt } from 'web3-core'
+
+@Console({
+  name: 'fund',
+  description: 'Manage the fund and relayers',
+})
+export class FundCommand {
+  constructor(
+    @Inject(networkConfig.KEY)
+    private readonly networkCfg: NetworkConfig,
+    private readonly fundingSvc: FundingService,
```

Comment:
How does the FundingService get the ContractKit with unlocked funding key?

Reply:
That happens in the way we include the BlockchainModule in the tools/app.module
But I have to tweak this a bit. Right now I need the tools module to act both as the Fund and as the Deployer and for dev envs that's probably ok but in prod those should be separate - I'm gonna refactor this today and also include the HSM linking.